app/views/media_objects/_metadata_display.html.erb

```diff
@@ -72,7 +72,8 @@ Unless required by applicable law or agreed to in writing, software distributed
       </h3>
     </div>
     <div class="card-body">
-      <%= @media_object.access_text %>
+      <%= @media_object.access_text %> <br />
+      <b>Lending Period:</b> <%= ActiveSupport::Duration.build(@media_object.lending_period).to_human_readable_s %>
```

Comment:
It looks like ActiveSupport::Duration will create a human readable string with `inspect`.  What do you think about using that?
```suggestion
      <b>Lending Period:</b> <%= ActiveSupport::Duration.build(@media_object.lending_period).inspect %>
```
If this is workable then we could avoid injecting a new method into Duration.

Reply:
I'm not sure it is workable. For example, it converts `2592000` (30 days in seconds) to "4 weeks and 2 days". And the conversions just get worse from there. "1 month, 2 weeks, 13 hours, 30 minutes, and 54 seconds" is the output for 45 days. So we would need extra parsing to convert them into better/clearer strings. Or override the behavior. 

To be clearer with the injected method though, I could change the name to something like `to_day_hour_s` since it is more specific than just making things human readable.